apps/website/src/content/docs/notificationmarker.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: NotificationMarker
+description: Shows a notification marker dot at the top right of the component.
+propsPath: '@itwin/itwinui-react/esm/core/NotificationMarker/NotificationMarker.d.ts'
```

Comment:
nit: `propsPath` doesn't need to be a variable

Reply:
Done (https://github.com/iTwin/iTwinUI/pull/1999/commits/5301a7f32352c0b22299664102bf1a712f9ebe53)